### PR TITLE
Cleaned up code that assumes v0 pods (container info)

### PR DESCRIPTION
**Pull request by solarkennedy**
We no longer need to roman ride!

### CODE REVIEW

**Review by sargun — APPROVED** [review]

File: executor/runner/runner_test.go [hunk 1]
```diff
@@ -329,15 +305,16 @@ func TestSendRedundantStatusMessage(t *testing.T) { // nolint: gocyclo
 		statusChan:  statusChan,
 	}
 
+	pod, _ := runtimeTypes.ContainerTestArgs()
 	task := Task{
 		TaskID:    taskID,
-		TitusInfo: taskInfo,
+		TitusInfo: nil,
```
[the remark this answers is not in the document]
Comment (solarkennedy) [reply executor/runner/runner_test.go:311]: No, I can erase that line (would default to nil)

File: executor/runtime/types/pod.go [hunk 2]
```diff
@@ -19,8 +18,7 @@ import (
 	podCommon "github.com/Netflix/titus-kube-common/pod"
 	resourceCommon "github.com/Netflix/titus-kube-common/resource"
 	"github.com/docker/distribution/reference"
-	units "github.com/docker/go-units"
-	"github.com/golang/protobuf/proto" // nolint: staticcheck
+	units "github.com/docker/go-units" // nolint: staticcheck
```
[the remark this answers is not in the document]
Comment (solarkennedy) [reply executor/runtime/types/pod.go:21]: none! (at least not anymore)

File: executor/runtime/types/types_test.go [hunk 3]
```diff
@@ -1,198 +1 @@
...
-	assert.True(t, c.TTYEnabled())
-}
-
-func TestOomScoreAdj(t *testing.T) {
-	taskID, titusInfo, resources, _, conf, err := ContainerTestArgs()
-	assert.NoError(t, err)
-	oomScore := int32(99)
-	titusInfo.OomScoreAdj = &oomScore
-
-	// TODO: This doesn't work on a V1 pod for some reason
```
[the remark this answers is not in the document]
Comment (solarkennedy) [reply executor/runtime/types/types_test.go:193]: Yes, covered in `TestPodContainerOomScoreAdj`

**Review by rgulewich — APPROVED** [review]
Wooooooooo!  Thank-you!

**Pull request merged**